otelcolbuilder/Makefile

```diff
@@ -93,20 +93,27 @@ _gobuild_debug:
 build: ensure-correct-builder-version
 # Since builder doesn't allow adding build tags, let's just skip the compilation
 # and run 'go build ...' by hand.
-	@$(MAKE) _builder SKIP_COMPILATION=true
+	@$(MAKE) generate-sources
 	@$(MAKE) _gobuild
 
 .PHONY: build-debug
 build-debug: ensure-correct-builder-version
 # Since builder doesn't allow adding build tags, let's just skip the compilation
 # and run 'go build ...' by hand.
-	@$(MAKE) _builder SKIP_COMPILATION=true
+	@$(MAKE) generate-sources
 	@$(MAKE) _gobuild_debug
 
 .PHONY: generate-sources
 generate-sources:
 	@$(MAKE) _builder SKIP_COMPILATION=true
 
+.PHONY: prepare-tag
+prepare-tag:
+	@[ "${TAG}" ] || ( echo ">> env var TAG is not set"; exit 1 )
+	$(eval ROOT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST)))))
```

Comment:
I wonder if it wouldn't be better to put this target in the root Makefile and avoid this path traversal.

Reply:
moved to main `Makefile`